datadog/provider_test.go

```diff
@@ -134,17 +136,36 @@ func initRecorder(t *testing.T) *recorder.Recorder {
 	return rec
 }
 
-func initAccProvider(t *testing.T, httpClient *http.Client) *schema.Provider {
+func testSpan(ctx context.Context, t *testing.T) (context.Context, func()) {
+	t.Helper()
+	span, ctx := tracer.StartSpanFromContext(
+		ctx,
+		"test",
+		tracer.SpanType("test"),
+		tracer.ResourceName(t.Name()),
+		tracer.Tag(ext.AnalyticsEvent, true),
+		tracer.Measured(),
+	)
+	// span.SetTag("version", tag)
```

Comment:
@bkabrda 👀  🏷️

Reply:
I would suggest doing the same thing as we do in https://github.com/DataDog/datadog-api-client-go/blob/master/tests/test_utils.go#L203